src/storage/index/LookupProcessor.cpp

```diff
@@ -10,6 +10,15 @@ namespace nebula {
 namespace storage {
 
 void LookupProcessor::process(const cpp2::LookupIndexRequest& req) {
+    if (executor_ != nullptr) {
+        executor_->add([req, this] () {
```

Comment:
Maybe we could move the req instead of copy it?

Reply:
> Maybe we could move the req instead of copy it?

Good point. changed to `by-reference capture` for lambda capture. But I'm not sure if I'll be released by `GraphStorageServiceHandler` . WDYT?

Reply:
I think capture by reference is ok.
![image](https://user-images.githubusercontent.com/13706157/105344877-7e241400-5c1e-11eb-920f-bad559619ec0.png)

Reply:
Got, Thanks.